vignettes/backtesting.Rmd

```diff
@@ -0,0 +1,460 @@
+---
+title: "Accurately backtesting forecasters"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Accurately backtesting forecasters}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+source(here::here("vignettes", "_common.R"))
```

Comment:
Since this is used only for vignette building, the `here` package would go under `Suggests` in DESCRIPTION. 

However, you don't need the path like this. When built, vignettes access this path. See: https://r-pkgs.org/vignettes.html#filepaths

Reply:
To iterate on building this vignette locally, you can do:

`devtools::build_rmd("vignettes/backtesting.Rmd")` which rebuilds the package, then renders the vignette.

Reply:
Vignettes having access to their own folder is a really unfortunate convention 😠 I prefer to iterate on vignette code in my REPL, not by rebuilding the whole vignette every time (and I know that caching speeds things up, but a built vignette doesn't let me interact with the objects like a REPL). I'm surprised this isn't an issue for RStudio users.

Not gonna waste time on figuring out a workaround, so I'll just do the relative path thing.